api/api.go

```diff
@@ -79,9 +98,12 @@ func routes(host string, router *mux.Router, dataStore DataStore, outputQueue Ou
 	api.router.HandleFunc("/filters/{filter_blueprint_id}/dimensions/{name}/options/{option}", api.removeFilterBlueprintDimensionOption).Methods("DELETE")
 
 	api.router.HandleFunc("/filter-outputs/{filter_output_id}", api.getFilterOutput).Methods("GET")
-	api.router.HandleFunc("/filter-outputs/{filter_output_id}", api.updateFilterOutput).Methods("PUT")
 	api.router.HandleFunc("/filter-outputs/{filter_output_id}/preview", api.getFilterOutputPreview).Methods("GET")
```

Comment:
Is preview just for internal(florence) users and hence should only be available if enablePrivateEndpoints is set to true?

Reply:
I don't think so - I believe this is the preview of the first 20 rows of data, so would be available for both